<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>120</number>
      </property>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>加载样式</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ChineseFont</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>WesternFont</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>FontSize</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Bold</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Italic</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Alignment</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>OutlineLevel</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>LeftIndent</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>RightIndent</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>FirstLineIndent</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>LineUnitBefore</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>LineUnitAfter</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>LineSpacing</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Shortcut</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>删除选中</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>添加项目</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>全部删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
